versioned_docs/version-8.0/guides/update-guide/connectors/introduction.md

```diff
@@ -0,0 +1,17 @@
+---
+id: introduction
+title: Introduction
```

Comment:
Should this be more verbose to allow easy discovery via search?

Reply:
@MaxTru, how about this?
```suggestion
title: Introduction Connectors updates
```

Reply:
Well is this only aboute SDK updates or in general "connector updates" (which could also be understood as "how to update your connector to a newer version when modeling")?

Reply:
Fair enough, I think it's just SDK updates. But I would put that up for question. 
@PreetiNirwal, @igpetrov, @nikku, thoughts?

Reply:
My understanding was SDK updates only. 
I suggest we think about connector templates separately and create different PRs to describe that.